doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -17,10 +17,36 @@
 <a name="4.1"></a>
 ## 4.1 Introduction
 
+This chapter will describe in detail the Kubernetes Reference Architecture in terms of the functional capabilities and how they relate to the Reference Model requirements, i.e. how the infrastructure profiles are delivered and described.
+
+Figure 4-1 below shows the architectural components that are described in the subsequent sections of this chapter.
+
+<p align="center"><img src="../figures/ch04_k8s_architecture.png" alt="Kubernetes Reference Architecture" Title="Kubernetes Reference Architecture" width="65%"/></p>
+<p align="center"><b>Figure 4-1:</b> Kubernetes Reference Architecture</p>
+
 <a name="4.2"></a>
 ## 4.2 Host OS
 
-> This chapter should describe the requirements for a host os to run the software stack of Reference Architecture 2.
+In order for a Host OS to be compliant with this Reference Architecture it must meet the following requirements:
```

Comment:
Can we assume that all deb/rpm compatible Linux distros have modern enough Kernel to run containers?

Reply:
Perhaps, but I think the kernel version should be covered by Table 4-1?

Reply:
Yes, but for me somehow Table 4-1 should be the consequence of the kernel and other requirements.

Reply:
Oh I see what you mean - yes fair enough.  I took the contents of Table 4-1 from the requirements of kubeadm - would calling that out make it clearer why I've put what I have in that table?

Reply:
Yes

Reply:
Done - thanks.